Add mapserver and corrections for mapfiles

ger-benjamin opened the pull request:
Add a mapserver to read our mapfiles and shapefiles

faselm commented:
Great work! I think we should add "COPY rasters /etc/rasters VOLUME /etc/rasters" in the Dockerfile as well to test the rasters.

ger-benjamin commented:
> Great work! I think we should add "COPY rasters /etc/rasters VOLUME /etc/rasters" in the Dockerfile as well to test the rasters.

Done, thanks.

> I am not familiar with docker but what is there seems to work. I fixed the test, i think it's good to go.

Sorry, I didn't see that. I've fixed the test also here.

I merge, thanks
